packages/compass-generative-ai/src/store/atlas-optin-reducer.ts

```diff
@@ -0,0 +1,310 @@
+import type { Action, AnyAction, Reducer } from 'redux';
```

Comment:
Can we add a test for this file? The signin-reducer already has one for reference.